Helper/Order.php

```diff
@@ -1173,13 +1173,13 @@ protected function deleteOrder($order)
      * Try to cancel the order. Covers the case when the payment was declined before authorization (blacklisted cc).
      * It is called upon rejected_irreversible hook.
      *
-     * @param $displayId
+     * @param $incrementId
+     * @param $quoteId
      * @return bool
      * @throws BoltException
      */
-    public function tryDeclinedPaymentCancelation($displayId)
+    public function tryDeclinedPaymentCancelation($incrementId, $quoteId)
```

Comment:
Can you rename `$quoteId` to `$immutableQuoteId` here and in the method body below, for consistency.